### ISSUE TITLE
bypassDnd not working

### ISSUE THREAD

**Issue by ashokkumar88** [notifee/react-native-notifee#206]
Hi,
I created a Android Channel with setting `bypassDnd: true`. But in the device Notification Settings the "Override Do Not Disturb" is still showing disabled. Seems the `bypassDnd` has no effect while creating a channel.

Notifee version: 0.15.1
Android 10

**Comment by mikehardy** [comment]
I don't believe the underlying API allows the channel creator to set it, no:

https://developer.android.com/reference/android/app/NotificationChannel#setBypassDnd(boolean)

This should be a read-only property on channel creation I think, with associated docs update - after a double-check on available platform docs and a quick test

**Comment by helenaford** [comment]
I think it's possible but need more steps before being able to do so by asking the user to grant the app permissions.
Looks like something we can implement so you can set the 'bypassDnd' on the channel.

References:
https://developer.android.com/reference/android/app/NotificationManager#isNotificationPolicyAccessGranted()
https://developer.android.com/reference/android/app/NotificationManager#setInterruptionFilter(int)

**Comment by vandana87** [comment]
Hi,
I have manually enabled dnd in my app settings and need custom notification with sound and vibration when the device is in do not disturb mode.
 Is it possible to get custom sound and vibration when we set device to do not disturb mode ?  
I am able to receive the notifications in Do not disturb mode but could not play custom sound and custom vibration when notification is received. I have enabled bypassDnd while creating channels as well.


below is the code I am using for creating channels:
```
 await notifee.createChannel({
     id: "example",
     name: "example",
     importance: AndroidImportance.HIGH,
     visibility: AndroidVisibility.PUBLIC,
     vibration: true,
     vibrationPattern: [300,500],
    sound:"chrip.wav",
   bypassDnd:true
   });
```
For displaying notification

```
const notification = {
  title: 'Message',
  body: "description",
  id: 'quickAction',
  android: {
      channelId:message.data.android_channel_id,
      pressAction: {
          id: 'default',
      },
      importance: AndroidImportance.HIGH,
      sound: message.data.soundname+'.wav',
      vibrationPattern:msgData1,
      visibility: AndroidVisibility.PUBLIC,
  },
  ios: {
      sound: message.data.soundname+'.wav',
      categoryId: 'quickActions',
  },

}


 try {
      await notifee.displayNotification(notification);
  }
  catch (e) {
      console.error(e);
  }
```
